src/Tools/MicroBuild/publish-assets.ps1

```diff
@@ -32,6 +32,25 @@ try
 {
     Write-Host "Starting PostTest script..."
 
+    # We need to remove 'refs/heads/' from the beginning of the string
+    $branchName = $branchName -Replace "^refs/heads/"
+    
+    # We also need to replace all instances of '/' with '_'
+    $branchName = $branchName.Replace("/", "_")
+
+    switch ($branchName)
+    {
+        "dev15-rc" { } 
+        default
+        {
+            if (-not $test)
+            {
+                Write-Host "Branch $branchName is not supported for publishing"
```

Comment:
The intent of the "list of branches to publish" was to explicitly mean you can fork a branch and build that without having to publish or remember to turn off publishing. We regularly have reasons to make private signed builds we don't want published. For example, to patch machines to do RPS validation. We want some private build to send to customers to validate a fix. @Pilchie might have had one or two other ideas too.

My point is we should just delete the "branch name not supported for publishing". If it's not supported, maybe give a warning, and that's it. I shouldn't have to do work to say I don't want publishing.